ui/pages/providers/add.js

```diff
@@ -155,6 +249,45 @@ export default function ProvidersAddDetails() {
       <Head>
         <title>Add Identity Provider - {kind}</title>
       </Head>
+      <header className='my-6 flex items-center justify-between'>
+        {/* SCIM key dialog */}
+        <Transition.Root show={keyDialogOpen} as={Fragment}>
```

Comment:
maybe we should componentized this - because we are using the same one for the other SCIM key dialog and also it looks very similar to the add group / user dialog too.

Reply:
Moved this out into a component in this PR, was fairly easy so I thought I may as well do it in the first go